aci/resource_aci_rtctrlattrp.go

```diff
@@ -92,6 +117,31 @@ func resourceAciActionRuleProfileImport(d *schema.ResourceData, m interface{}) (
 	if err != nil {
 		return nil, err
 	}
+
+	// rtctrlSetTag - Beginning Import
+	checkDns := make([]string, 0, 1)
+
+	setRouteTagDn := rtctrlAttrP.DistinguishedName + fmt.Sprintf("/"+models.RnrtctrlSetTag)
+
+	checkDns = append(checkDns, setRouteTagDn)
+
+	err = checkTDn(aciClient, checkDns)
+	if err == nil {
```

Comment:
I don't think you need the checkDns as it is a children of the parent you are manipulating.